openerp/models.py

```diff
@@ -2160,6 +2160,14 @@ def read_group(self, cr, uid, domain, fields, groupby, offset=0, limit=None, con
             for d in fetched_data:
                 d.update(data_dict[d['id']])
 
+        selectionfields = [gb['field'] for gb in annotated_groupbys if gb['type'] == 'selection']
+        if selectionfields:
+            selections_values = {}
+            for selection in selectionfields:
+                selections_values[selection] = dict(self._fields[selection].selection)
```

Comment:
I will wait @rco-odoo review then :). It was the point in the first place anyway : D.